adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -103,37 +109,85 @@ - (BOOL)isSDKInitialized {
   return [[VungleSDK sharedSDK] isInitialized];
 }
 
-- (BOOL)isAdCachedForPlacementID:(nonnull NSString *)placementID {
-  return [[VungleSDK sharedSDK] isAdCachedForPlacementID:placementID];
+- (BOOL)isAdCachedForPlacementID:(nonnull NSString *)placementID withDelegate:(nonnull id<GADMAdapterVungleDelegate>)delegate {
+  GADMAdapterVungleAdType adType = [delegate adapterAdType];
+  if (adType != GADMAdapterVungleAdTypeBanner && adType != GADMAdapterVungleAdTypeShortBanner && adType != GADMAdapterVungleAdTypeLeaderboardBanner) {
+    return [[VungleSDK sharedSDK] isAdCachedForPlacementID:placementID];
```

Comment:
What would Vungle return if calling this method with a banner placement here without a size? Return true if any size is cached for that banner?

Reply:
@ericleich Yes, it will return true if any size is cached for that banner. Thanks a lot O(∩_∩)O